packages/model-viewer/README.md

```diff
@@ -41,6 +41,31 @@ It can also be used directly from various free CDNs such as [unpkg.com](https://
 For more detailed usage documentation and live examples, please visit our docs
 at [modelviewer.dev](https://modelviewer.dev)!
 
+### Important note about versions
+Our goal for `<model-viewer>` is to be a consistent, stable part of your web
+platform while continuing to deliver cutting-edge features. We’ll always try
+to minimize breaking changes, and to keep the component backwards compatible.
+See our [guide to contributing](../../CONTRIBUTING.md#Stability) for more
+information on backwards compatibility.
+
+For your production site you may want the extra stability that comes by
+pinning to a specific version, and upgrading on your own schedule (after
+testing).
+
+If you’ve installed via [NPM](https://npmjs.org), you’re all set - you’ll only
+upgrade when you run [`npm update`](https://docs.npmjs.com/cli/update.html).
+
+If you’re using [unpkg.com](https://unpkg.com), you can pin to a specific
+version by specifying it in the URL (the sample below pins to version 0.10.0):
+
+```html
+<!-- Loads <model-viewer> for modern browsers: -->
+<script type="module"
+    src="https://unpkg.com/@google/model-viewer@0.10.0/dist/model-viewer.js">
+</script>
```

Comment:
I'm a little nervous about keeping a fixed version in an example in the README because we need to remember to update it with each release. I know it's tricky to convey the concept without doing that, but also I know from our cookbook results that folks are just going to blindly copy/paste example snippets like this.

Reply:
Agreed! I had the same problem. For a bit I had a <version> marker in the URL - I could switch back to that if you think it'd be better?

Reply:
Personally I would be happier if we included a generic snippet that will fail. Maybe we can strive towards some automation that will auto-update this part of the README for us? Github Actions seems like a very suitable solution for this.